src/Algebra/Module/Construct/Zero.agda

```diff
@@ -10,61 +10,62 @@
 
 {-# OPTIONS --without-K --safe #-}
 
-module Algebra.Module.Construct.Zero where
+open import Level
+
+module Algebra.Module.Construct.Zero {c ℓ : Level} where
 
 open import Algebra.Bundles
 open import Algebra.Module.Bundles
-open import Data.Unit
-open import Level
+open import Data.Unit.Polymorphic
 
 private
   variable
     r s ℓr ℓs : Level
 
-leftSemimodule : {R : Semiring r ℓr} → LeftSemimodule R 0ℓ 0ℓ
+leftSemimodule : {R : Semiring r ℓr} → LeftSemimodule R c ℓ
```

Comment:
Is there a reason all of these have level indices `c ℓ`
whereas in `Algebra.Construct` all of the definition return
constructs using `ℓ ℓ`?

Reply:
Thanks for the eyes and the question! Now I'm asking myself the same question. I just changed `Algebra.Construct` to use `c ℓ` as well. When `make clean && make test` finish, I'll update this pull request.